core/tasks.py

```diff
@@ -12,40 +13,49 @@
 from chronam.core.models import Batch, OcrDump
 from chronam.core.batch_loader import BatchLoader
 
-logger = logging.getLogger(__name__)
+_logger = logging.getLogger(__name__)
 
+@task
+def load_page_in_cache(url):
+    try:
+        _logger.debug("Requesting page [%s]" % url)
+        r = requests.get(url)
+        _logger.debug("status %s" % r.status_code)
+        r.raise_for_status() #throw an error if status is not OK
+    except Exception, e:
+        _logger.exception("Unable to request page [%s]: %s" % (url, e))
 
 @task
 def process_coordinates(batch_dir):
     try:
         batch_loader = BatchLoader()
         batch_loader.process_coordinates(batch_dir)
-        logger.info("processed batch %s", batch_dir)
+        _logger.info("processed batch %s", batch_dir)
     except Exception, e:
-        logger.exception("unable to process batch %s" % batch_dir)
+        _logger.exception("unable to process batch %s" % batch_dir)
```

Comment:
I might have voted for changing this variable name to be a separate commit which could be easily reviewed. While we're doing this, we should fix all of the instances where string interpolation is being forced before calling a logger. `pylint` records this using the `logging-not-lazy` warning:

```
W: 85, 8: Specify string format arguments as logging function parameters (logging-not-lazy)
```